dev-docs/prebid-1.0-API.md

```diff
@@ -65,21 +61,38 @@ The input to `pbjs.setConfig` must be JSON (no JavaScript functions are allowed)
 {% highlight js %}
 
 pbjs.setConfig({
-    "currency": {
-        // Enables currency feature -- loads the rate file
+    currency: {
         "adServerCurrency": "JPY",
-        // Allows the publisher to override the default rate file
-        "conversionRateFile": "url"
+        "conversionRateFile": "<url>"
     },
-    "debug": true, // Previously `logging`
-    "s2sConfig": { ... },
-    "priceGranularity": "medium",
-    "enableSendAllBids": false, // Default will be `true` as of 1.0
-    "bidderSequence": "random",
-    "bidderTimeout": 700, // Default for all requests.
-    "publisherDomain": "abc.com", // Used for SafeFrame creative.
-    "pageOptions": { ... },
-    "sizeConfig": { ... }
+    debug: true, // Previously `logging`
+    s2sConfig: { ... },
+    priceGranularit": "medium",
+    enableSendAllBids: false, // Default will be `true` as of 1.0
+    bidderSequence: "random", // Default is random
+    bidderTimeout: 700, // Default for all requests.
+    publisherDomain: "abc.com", // Used for SafeFrame creative.
+    pageOptions: { ... },
+    sizeConfig: { ... },
+    cache: {url: "<prebid cache url>"}
+});
+
+{% endhighlight %}
+
+## No More Default Endpoints
+
+In Prebid 0.x there were defaults for the Prebid Server endpoints and the video cache URL. With 1.0, these defaults have been removed to be vendor neutral, so all publisher pages must define them via pbjs.setConfig(). The same functionality as 0.x may be achieved with this setConfig: 
```

Comment:
I'd revise the last phrase to "may be achieved as shown below" to avoid using the method name in prose right before just showing a code sample anyway.

Reply:
updated